hack/verify-flags/exceptions.txt

```diff
@@ -1,4 +1,87 @@
 Vagrantfile:      node_ip = $node_ips[n]
+_output/release-stage/full/kubernetes/Vagrantfile:      node_ip = $node_ips[n]
```

Comment:
Please remove all those "_output" lines.

(You probably need to do make clean before running the flag script).